scripts/build_docker

```diff
@@ -34,17 +34,17 @@ shift $(($OPTIND - 1))
 # Set build parameters
 
 case ${TAG} in
-    amd64)
+    amd64*)
 	BUILD_OPTS='-b'                  # Build all binary packages
 	RUN_TESTS='runtests tests'       # Run tests on build arch
 	;;
-    i386)                                # Machine arch: i386
+    i386*)                               # Machine arch: i386
 	BUILD_OPTS="-a i386"             # - Set machine arch
 	;;&
-    armhf|raspbian)                      # Machine arch: armhf
+    armhf*|raspbian*)                    # Machine arch: armhf
 	BUILD_OPTS="-a armhf"            # - Set machine arch
 	;;&
-    i386|armhf|raspbian)                 # Cross-compile/foreign arch
+    i386*|armhf*|raspbian*)              # Cross-compile/foreign arch
 	BUILD_OPTS+=" -B"                # - Only build arch binary packages
 	BUILD_OPTS+=" -d"                # - Root fs missing build deps; force
```

Comment:
Yes, I found it was not possible to build with `-D` because of the zeromq and czmq packages.
`-d` works fine.

For the same reason I made a change to debian/rules.in https://github.com/machinekit/machinekit/blob/master/debian/rules.in#L246, which prevents dh_shlibdeps throwing an error at those packages.

Once we get the main code able to build using the new zeromq code, it will go away.